docs/chapters/licensing/index.rst

```diff
@@ -0,0 +1,25 @@
+Licensing
+=========
+
+Each source code file needs to mention what license applies and who owns
+the copyright in the file header. The purpose of this is to make it
+absolutely clear, for both humans and tooling, which license it is that
+applies to that particular file even when the file is moved to another
+project. In order to achieve this, it has been decided that the header
+should include:
```

Comment:
I think we should write this in wordings that describe what the user should do, and ofc have rationales for that in separate paragraphs. So I don't think the docs read nicely if we say things like "it has been decided...", it could perhaps be "to cover the above concerns the header should include:".

It's nitpicky-ish perhaps but I think we should keep this in mind throughout the docs.